packages/gatsby-cli/src/create-cli.ts

```diff
@@ -34,8 +34,8 @@ const handlerP = (fn: (args: yargs.Arguments) => void) => (
 }
 
 function buildLocalCommands(cli: yargs.Argv, isLocalSite: boolean): void {
-  const defaultHost = `localhost`
-  const defaultPort = `8000`
+  const defaultHost = process.env.DEFAULT_GATSBY_HOST || `localhost`
+  const defaultPort = process.env.DEFAULT_GATSBY_PORT || `8000`
```

Comment:
```suggestion
  const defaultHost = process.env.GATSBY_DEFAULT_HOST || `localhost`
  const defaultPort = process.env.GATSBY_DEFAULT_PORT || `8000`
```

Don't we use our prefix at the beginning always?

Reply:
The reason I didn't was so it wasn't exposed in the runtime

Reply:
ah, okay, gotcha 👍 Still wonder what other env vars for the CLI we have so that it's coherent